extension.json

```diff
@@ -98,7 +77,7 @@
               "configId",
               "name"
             ],
-            "additionalProperties": false
+            "additionalProperties": true
```

Comment:
Do you mind logging a JIRA issue to remind us to switch it back before releasing v1?

Reply:
https://jira.corp.adobe.com/browse/CORE-40600